### PR TITLE
My pull request

### PR DESCRIPTION
**Pull request by sarkarBetaV**
Changed branch name
More specific details about the tech stacks used.
fixed the MIT License error
Changed the Logo image.

### CODE REVIEW

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @sarkarBetaV 🙋‍♂️
Good job working on your hello Microverse project. You have done incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Your project is professional ✅
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

- [ ] I suggest that you consider using a descriptive branch name like `contact-form` to ensure the branch visitors have a structural understanding of the features implemented in that particular branch ☺️.
- [ ] Please, I suggest that you provide a descriptive PR description and title. This to ensure that you provide the structural understand of the features implemented in that PR ☺️.


## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md [hunk 1]
```diff
@@ -1 +1,283 @@
...
+
+After you're finished please remove all the comments and instructions!
+
+For more information on the importance of a professional README for your repositories: https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/readme_best_practices.md
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
[the remark this answers is not in the document]
Comment (sarkarBetaV) [reply README.md:34]: added a different logo

File: README.md [hunk 2]
```diff
@@ -1 +1,283 @@
...
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
[the remark this answers is not in the document]
Comment (sarkarBetaV) [reply README.md:280]: can u plz suggest how would i fix it?

File: README.md
Comment (joskalenda) [inline README.md:1]: - [ ] You have well created the MIT.md file but I think you have forgotten to update your readme file on the go, currently, when clicking the mit link, it redirects the user to the /LICENSE path a file that does not exist, please change your link @sarkarBetaV

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @sarkarBetaV , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ]... ✅ Green linters


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sarkarBetaV** [comment]
Hello @joskalenda I am struggling with the MIT license part, kindly guide me how do i fix it

File: README.md
Comment (sarkarBetaV) [reply README.md:1]: Hi @joskalenda Can u pls elaborate how do i solve this MIT license issue?

**Review by Munsa1 — APPROVED** [review]
Hi @sarkarBetaV, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Fixed Broken MIT License Link :heavy_check_mark: 
 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sarkarBetaV** [comment]
merging the pull request

**Pull request merged**